README.md

```diff
@@ -6,9 +6,9 @@
 A JavaScript project for the Leaderboard list app, using webpack and ES6 features, that consumes the Leaderboard API using JavaScript async and await.
 
 ## Built With
-- Major languages: HTML, JavaScript
-- Frameworks:
-- Technologies used: Linters, Git, npm, Webpack
+- Major languages: HTML, SCSS, JavaScript
+- Frameworks: Bootstrap-v5
+- Technologies used: Linters, Git, npm, Webpack, Sass
 
```

Comment:
- [ ] OPTIONAL
I think you should change the header title _Authors_ to _Author_ in your ReadMe on _line 22_, as this project is worked on by sole contributor. 

- [ ] OPTIONAL
Kindly replace the text **Author1** with your full name on _line 24_, so as to make your ReadMe more professional